storybook/.storybook/nuxeo-demo-theme.js

```diff
@@ -4,11 +4,200 @@ import '@nuxeo/nuxeo-ui-elements/nuxeo-button-styles.js';
 const template = html`
   <custom-style>
     <style is="custom-style" include="nuxeo-button-styles">
+      [hidden] {
+        display: none !important;
+      }
+
+      h1 {
+        font-weight: 400;
+        font-size: 2rem;
+        letter-spacing: 0;
+        line-height: 2.5rem;
+      }
+
+      h2 {
+        font-weight: 400;
+        font-size: 1.75rem;
+        letter-spacing: 0;
+        line-height: 2.25rem;
+      }
+
+      h3 {
+        font-weight: 400;
+        font-size: 1.25rem;
+        letter-spacing: 0;
+        line-height: 1.625rem;
+      }
+
+      h4 {
+        font-weight: 600;
+        font-size: 1rem;
+        letter-spacing: 0;
+        line-height: 1.375rem;
+      }
+
+      h5 {
+        font-weight: 700;
```

Comment:
The `font-weight` for `h4` and `h6` is `600` but for `h5` is `700`?

Reply:
Yes, it's a design decision stated on our [typography guidelines](https://nuxeowiki.atlassian.net/wiki/spaces/DT/pages/889815048/Typography). Thanks for raising this question because I completely forgot to mention this wiki page.